core/deployment/src/main/java/io/quarkiverse/operatorsdk/deployment/helm/DisableDefaultHelmListener.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkiverse.operatorsdk.deployment.helm;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.dekorate.WithSession;
+import io.dekorate.kubernetes.config.BaseConfigFluent;
+import io.dekorate.kubernetes.config.Configurator;
+
+public class DisableDefaultHelmListener extends Configurator<BaseConfigFluent<?>> implements WithSession {
```

Comment:
What's the point of this class?

Reply:
added comment. it is to disable default helm generator